src/main.js

```diff
@@ -40,10 +40,17 @@ async function handleDisplayWeatherBySearch() {
 
   if (splitText.length === 0) return;
 
+  if (splitText.startsWith(",")) {
+    splitText = splitText.replace(/^[^a-z][^A-Z]*/, "");
```

Comment:
Could you please explain what it does?

Reply:
if it starts with a comma it erases all characters until it finds a letter, but i'm getting an error again when inputting .,some city.i have not covered that.i don't like it that much...i will redo it

Reply:
How about: `input string` => split by comma => filter out all empty arrays => trim whitespace on each item(word) => join by comma => `query string`.

I am afraid that the regex would reject any Czech characters for example.. or Greek and Russian alphabet..

Reply:
Moreover, the API does some own validation so if the user inserts some gibberish, it will either return `[]` for not finding anything or a `400` error. In either case, `updateWeather` should not be called.

Reply:
This is for the search text value mostly...can i take the Name of the city from the response and pass it to the searchText value?but in the language it is inputted

Reply:
If i can take the system's language and search response by it,would i be able to do that?...it would look exactly like the api's response name and it would save the hassle of leaving dots or - where they are supposed to be

Reply:
No,this would not take into account state code or country...anyway,i will redo everything